defaults/main.yml

```diff
@@ -16,6 +16,7 @@ yum_cron_hourly_update_message: yes
 yum_cron_hourly_download_updates: yes
 yum_cron_hourly_apply_updates: yes
 yum_cron_hourly_random_sleep: 5
+yum_cron_hourly_base_options: []
```

Comment:
Hmm... there's a commented out example immediately below. It's been a while since I've worked with this, so I don't recall why this is commented out but I think the intent is to demonstrate this to users and then they can actually set if desired. I don't think we should set an empty value because there may be times we want to set a check on "is defined" rather than "is empty".

So can you delete this line or explain why it needs to be initialized?